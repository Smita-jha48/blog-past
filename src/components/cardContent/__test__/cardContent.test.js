import { render } from '@testing-library/react';
import CardContent from '../cardContent';

describe('CardContent', () => {
  it('should render correctly', () => {
    const mockProps = {
        title: "xyz",
        description: "tjxnzh",
    }
    const { asFragment } = render(<CardContent {...mockProps}/>);
    expect(asFragment()).toMatchSnapshot();
  });
});