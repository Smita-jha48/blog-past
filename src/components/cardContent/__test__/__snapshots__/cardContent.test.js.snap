// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardContent should render correctly 1`] = `
<DocumentFragment>
  <div
    class="middle"
  >
    <p
      class="title"
    >
      xyz
    </p>
    <p
      class="about"
    >
      tjxnzh
    </p>
  </div>
</DocumentFragment>
`;
